### ISSUE TITLE
Please enable graphql 14.0.0 as peer dependency.

### ISSUE THREAD

**Issue by jonaskello** [graphql/graphiql#866]
The graphql package is now available in a major version as of 14.0.0 Yarn will spit out a lot of errors becuase I override with 14.0.0. So this request is similar to #247 but for 14.0.0.

**Comment by jonaskello** [comment]
Having upgraded to graphiql 1.13.2 things get a little bit better but some packages are still missing graphql 14.0.0 as peer dep. These are the warnings I get using graphiql 1.13.2:

```
warning graphiql > codemirror-graphql > graphql-language-service-interface@1.3.2" has incorrect peer dependency "graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0".
warning graphiql > codemirror-graphql > graphql-language-service-parser@1.2.2" has incorrect peer dependency "graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0".
warning graphiql > codemirror-graphql > graphql-language-service-interface > graphql-config@2.0.1" has incorrect peer dependency "graphql@^0.11.0 || ^0.12.0 || ^0.13.0".
warning graphiql > codemirror-graphql > graphql-language-service-interface > graphql-language-service-types@1.2.2" has incorrect peer dependency "graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0".
warning graphiql > codemirror-graphql > graphql-language-service-interface > graphql-language-service-utils@1.2.2" has incorrect peer dependency "graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0".
warning graphiql > codemirror-graphql > graphql-language-service-interface > graphql-config > graphql-import@0.4.5" has incorrect peer dependency "graphql@^0.11.0 || ^0.12.0 || ^0.13.0".
```

**Comment by acao** [comment]
hey @jonaskello! we were planning on putting out the rest of the releases to resolve this this weekend.

**Comment by acao** [comment]
fixed in 0.9

**Comment by jonaskello** [comment]
Yes, no more warnings :-). Thanks for fixing this!
